### PR TITLE
add notes for appearance in old ff

### PR DESCRIPTION
**Pull request by sergey-kozlov**

### CODE REVIEW

File: features-json/css-appearance.json [hunk 1]
```diff
@@ -261,7 +261,8 @@
   "notes":"",
   "notes_by_num":{
     "1":"The appearance property is supported with the `none` value, but not `auto`. Webkit, Blink, and Gecko browsers also support additional vendor specific values.",
-    "2":"Microsoft Edge and IE Mobile support this property with the `-webkit-` prefix, rather than `-ms-` for interop reasons."
+    "2":"Microsoft Edge and IE Mobile support this property with the `-webkit-` prefix, rather than `-ms-` for interop reasons.",
+    "3":"-moz-appearance:none on a combobox not remove the dropdown arrow"
```
Comment (cvrebert) [inline features-json/css-appearance.json]: HTML doesn't have comboboxes. Do you mean `<select>`?
Also, this is missing a verb; should be "does not".

Comment (Fyrd) [inline features-json/css-appearance.json]: @sergey-kozlov can you address this? ^

Comment (sergey-kozlov) [inline features-json/css-appearance.json]: Sorry, I do not speak English.
"-moz-appearance:none doesn't remove the dropdown arrow in select tag" better? Added commit.

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**